Added doc content for workflow (#17)
scbd/clearing-house-documentation: components/ircc/workflow/request-body.md

The request body should be a JSON object containing the following fields:

| Field           | Type          | Description   | Required |
| --------------- | ------------- | ------------- | -------- |

> | `document`      | Object        | The schema object that represents the document being submitted. For example, the [IRCC schema](/ircc/schema). | Yes      |

Reviewer: ```suggestion
| `document`      | Object        | The schema object that represents the body of the record type. For example, the [IRCC schema](/ircc/schema). | Yes      |
```

> | `additionalInfo`| String        | Additional information related to the document. This field can hold a maximum of 1000 characters.    | No       |

Reviewer: this field is available only when a use with NAU role is submitting a request to the PA to publish a record.
